CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+The latest version is considered WIP and it is a subject of change. All other
+versions are frozen. To freeze current version all files are copied to a new
+version directory, and then changes are introduced.
+
+
+## [0.4.1] WIP
```

Comment:
let's not have WIP anything in these repos, let's stay clean and only have actual releases

Reply:
ok